build/platform-mingw_nt.mk

```diff
@@ -1,7 +1,7 @@
 include $(SRC_PATH)build/x86-common.mk
 SHAREDLIB_DIR = $(PREFIX)/bin
 SHAREDLIBSUFFIX = dll
-SHAREDLIBSUFFIXVER=$(SHAREDLIBSUFFIX)
+SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX)
```

Comment:
please also export this variable together:
SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX)